symfony/framework-bundle/3.3/manifest.json

```diff
@@ -13,7 +13,9 @@
     },
     "env": {
         "APP_ENV": "dev",
-        "APP_SECRET": "%generate(secret)%"
+        "APP_SECRET": "%generate(secret)%",
+        "#APP_TRUSTED_PROXIES": "127.0.0.1,127.0.0.2",
+        "#APP_TRUSTED_HOSTS": "localhost,example.com"
```

Comment:
I would remove the `APP_` prefix

Reply:
Ok, but other vars below are prefixed by `APP` (and I think it's a common practice to prefix env vars: https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#adding-custom-environment-variables)

Reply:
I think it makes sense for "core" env var names as they are quite generic (DEBUG, ENV, ... would be too generic), but for these ones, I don't think we should fear a conflict.